bytebuffer-collections/src/main/java/io/druid/collections/spatial/RTree.java

```diff
@@ -128,8 +128,8 @@ private Node buildRoot(boolean isLeaf)
   {
     float[] initMinCoords = new float[numDims];
     float[] initMaxCoords = new float[numDims];
-    Arrays.fill(initMinCoords, -Float.MAX_VALUE);
-    Arrays.fill(initMaxCoords, Float.MAX_VALUE);
+    Arrays.fill(initMinCoords, Float.NEGATIVE_INFINITY);
```

Comment:
I think it should be ok. I didn't check the full code around RTree in druid, but usually the min/max coordinates of an RTree node represents a minimum range covering all coordinates of the children nodes. It is used for early pruning when traversing the tree. So, I think the min/max coordinates of the root node shouldn't used for any computation.